Client/Application/AppDelegate+SyncSentTabs.swift

```diff
@@ -30,46 +30,12 @@ class AppSyncDelegate: SyncDelegate {
         self.logger = logger
     }
 
-    func displaySentTab(for url: URL, title: String, from deviceName: String?) {
+    func openSendTabs(for urls: [URL]) {
         DispatchQueue.main.async {
             if self.app.applicationState == .active {
-                let object = OpenTabNotificationObject(type: .switchToTabForURLOrOpen(url))
-                NotificationCenter.default.post(name: .OpenTabNotification, object: object)
-                return
-            }
-
-            // check to see what the current notification settings are and only try and send a notification if
-            // the user has agreed to them
-            UNUserNotificationCenter.current().getNotificationSettings { settings in
-                if settings.alertSetting != .enabled {
-                    return
-                }
-
-                let notificationContent = UNMutableNotificationContent()
-                let title: String
-                if let deviceName = deviceName {
-                    title = String(format: .SentTab_TabArrivingNotification_WithDevice_title, deviceName)
-                } else {
-                    title = .SentTab_TabArrivingNotification_NoDevice_title
-                }
-                notificationContent.title = title
-                notificationContent.body = url.absoluteDisplayExternalString
-                notificationContent.userInfo = [SentTabAction.TabSendURLKey: url.absoluteString, SentTabAction.TabSendTitleKey: title]
-                notificationContent.categoryIdentifier = "org.mozilla.ios.SentTab.placeholder"
-
-                // `timeInterval` must be greater than zero
-                let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 0.1, repeats: false)
-
-                // The identifier for each notification request must be unique in order to be created
-                let requestIdentifier = "\(SentTabAction.TabSendCategory).\(url.absoluteString)"
```

Comment:
I think `SentTabAction.TabSendCategory` is not used anymore and can be removed entirely?